src/app/components/text-input-field.tsx

```diff
@@ -126,14 +131,14 @@ export function TextInputField({
           height="24px"
           id={name}
           isDisabled={isDisabled}
-          p="none"
-          placeholder={placeholder || capitalize(name)}
+          p={variant === 'mnemonic' ? '10px' : 'none'}
+          placeholder={placeholder === 'none' ? undefined : placeholder || capitalize(name)}
           spellCheck="false"
```

Comment:
This was set as `type="input"` but I've changed the default to be `text`. Added this reminder in case it breaks something else